extensions/amp-live-list/0.1/amp-live-list.js

```diff
@@ -861,6 +921,11 @@ export class AmpLiveList extends AMP.BaseElement {
         this.viewport_.getScrollTop() + this.viewport_.getSize().height;
   }
 
+  isElementAboveViewport_(element) {
+    return this.viewport_.getLayoutRect(element).bottom <
+        this.viewport_.getScrollTop();
```

Comment:
Not sure if `getScrollTop()` would be accurate here ( given the whole padding-top we add for viewer header). I suggest either:
1- do `viewport.getRect().bottom < element.getLayoutBox().top` or
2- check if `element.getIntersectionChangeEntry()` has `intersectionRatio` of 1.
(I assume element is an amp element?)

Reply:
@aghassemi element here can be a normal html tag (doesn't have to be a component) so i would probably need to do option 1.

Reply:
sgtm

Reply:
@aghassemi `element.getLayoutBox()` doesn't exist correct?